packages/@pollyjs/adapter-node-http/src/index.js

```diff
@@ -120,6 +121,11 @@ export default class HttpAdapter extends Adapter {
       const module = modules[moduleName];
       const { request, get, globalAgent } = module;
 
+      this[moduleName] = {
```

Comment:
nit: I'll probably end up renaming this but don't feel like blocking the PR on it :)